.github/workflows/package-client.yml

```
@@ -10,6 +10,12 @@ on:
         description: The version to use
         type: string
         required: true
+      version_patch_run_id:
+        description: |
+          The run id where the version.patch artifact was uploaded.
+          if not provided, the workflow will generate the patch for itself.
```

Do you mean by itself?
```suggestion
          If not provided, the workflow will generate the patch by itself.
```